src/styles/blocks/logo.scss

```diff
@@ -0,0 +1,10 @@
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+
+  &__image {
+    height: 100%;
+    width: 100%;
+  }
+}
```

Comment:
Consider using a nested structure for BEM elements only when necessary. If '.logo__image' does not have modifiers or elements of its own, it's cleaner to not nest it under '.logo' to keep the CSS flat and more readable.